books/free-programming-books-ta.md

```diff
@@ -32,6 +34,16 @@
 * [எளிய தமிழில் Computer Vision](https://freetamilebooks.com/ebooks/computer_vision/) - இரா.அசோகன் (PDF)
 
 
+### Deep Learning
+
+* [எளிய தமிழில் Deep Learning](https://freetamilebooks.com/ebooks/learn_deep_learning_in_tamil/) - து. நித்யா (PDF)
```

Comment:
Put this book iin the Machine Learing category

Reply:
Sure, I'll do it right away

Reply:
@eshellman Please review it now